fable-test-app/src/components/NextHoliday.tsx

```diff
@@ -1,19 +1,25 @@
 import React from 'react';
-import { Text } from '../components';
+import { Text, Heading } from '../components';
+
+import { Provinces } from '../utils/constants';
 
 interface NextHolidayProps {
-  display?: 'banner' | 'table';
+  display?: 'banner' | 'table' | 'homepage';
   isCurrentHoliday? : boolean;
   nextHoliday: {
     date: string;
     nameEn: string;
   } | null;
+  federal?: boolean;
+  observedIn?: Provinces[];
```

Comment:
I like this one better for clarity, what do you think?
```suggestion
  provincesObservedIn?: Provinces[];
```